blog/docs/articles/aws_to_func_migration.md

```diff
@@ -0,0 +1,103 @@
+# Migrating Functions from AWS Lambda to Knative Functions using Golang
+
+**Author: Matthias Weßendorf, Senior Principal Software Engineer @ Red Hat**
+
+_In a [previous post](/blog/articles/consuming_s3_data_with_knative){:target="_blank"} we discussed the consumption of notifications from an AWS S3 bucket inside a Knative Function. This post will describe the migration from a AWS Lambda Function, receiving S3 notifications, to Knative Functions._
+
+With Serverless Functions one of the common use-cases is to execute custom code based on an event trigger, like a notification from the AWS S3 service. With AWS Lambda you can run those programs on Amazon's cloud offerings, but running the code on your own data-center is much harder.
+
+## A simple AWS Lambda function for AWS S3
+
+Taking a look at their [sample repository](https://github.com/aws/aws-lambda-go/blob/main/events/README_S3.md){:target="_blank"} shows a minimal, yet complete function for receiving AWS S3 event notifications. Lets take a look at their code:
+
+```go
+// main.go
+package main
+
+import (
+	"fmt"
+	"context"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-lambda-go/events"
+)
+
+func handler(ctx context.Context, s3Event events.S3Event) {
+  	for _, record := range s3Event.Records {
+      		s3 := record.S3
+      		fmt.Printf("[%s - %s] Bucket = %s, Key = %s \n", record.EventSource, record.EventTime, s3.Bucket.Name, s3.Object.Key)
+  	}
+}
+
+
+func main() {
+	// Make the handler available for Remote Procedure Call by AWS Lambda
+	lambda.Start(handler)
+}
+```
+
+You see a two functions here, the `handler` function for the custom application logic and a `main` function which calls some AWS Lambda APIs and registers the custom handler with it. The signature of the `handler` references the standard `Context` and an `S3Event` from the AWS Lambda SDK. In order to be able to run the function you need a two vendor specific dependencies. The `main` function is also not specific to the actual program, but more technically needed in order have the custom `handler` receive events. 
+
+
+## A much simpler Knative Function for AWS S3
+
+In the [previous post](/blog/articles/consuming_s3_data_with_knative){:target="_blank"} we already discussed how to consume notifications from AWS Lambda in an on-premise cluster using Knative Eventing, but lets take a look at the code again:
+
```

Comment:
It's not clear how user can create / bootstrap Knative Function. I.e. I though I need to copy/paste the code from blog etc.
I'd suggest to add a snippet of `func create` before the `main.go` code block.

Reply:
this is covered in the previous blog as well